templates/systemd-service.template

```diff
@@ -19,7 +19,12 @@ RestartSec=2s
 
 ExecStartPre=-/usr/bin/docker stop {{ service_name }}
 ExecStartPre=-/usr/bin/docker rm {{ service_name }}
-ExecStartPre={{ service_pre_command }}
+
+{% if service_pre_command == "" %}
+    print("Enter path to pull docker")
+{% else %}
```

Comment:
If you directly check if the variable is defined, there won't be a need for an `else`.